jdaviz/configs/imviz/helper.py

```diff
@@ -215,6 +215,48 @@ def link_data(self, **kwargs):
         """
         link_image_data(self.app, **kwargs)
 
+    def get_link_type(self, data_label_1, data_label_2):
+        """Find the type of ``glue`` linking between the given
+        data labels. A link is bi-directional. If there are
+        more than 2 data in the collection, one of the given
+        labels should be the reference data or look-up will fail.
```

Comment:
@bmorris3 - does the rotation work remove some of this flexibility (by forbidding mixed linking)?  If we might need to change or remove this, should we make it private in the meantime?

Reply:
I think this method is useful with or without the rotation work. It's likely the case that it will be redundant with the link type attribute after image rotation is merged and WCS-linked in Imviz, but I think it's handy for debugging.

Reply:
Do you think it should be public or private in the meantime?

Reply:
I don't see why it should be private. The information is so useful in the meantime that I think it should be public. If it comes a day we don't need it anymore, then we can deprecate it.